server/tests/oauth/index.spec.js

```diff
@@ -0,0 +1,48 @@
+import app from '<server>/app';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+chai.use(chaiHttp);
+
+const { expect } = chai;
+describe('Oauth signup', () => {
+  it('should successfully authenticate via facebook', (done) => {
+    chai
+      .request(app)
+      .get('/api/v1/auth/facebook')
+      .end((err, res) => {
+        expect(res.status).to.be.equal(200);
```

Comment:
Please check the other parameters of the `res.body`

Reply:
I tried doing this but res.body was empty object( i.e {}). This route is being responded to by facebook OAuth page and its res contains status and other objects.

I decided to test for the text object to be a string